x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/arithmetic/ArithmeticFunction.java

```diff
@@ -65,7 +66,7 @@ protected ProcessorDefinition makeProcessorDefinition() {
     public String name() {
         StringBuilder sb = new StringBuilder();
         sb.append("(");
-        sb.append(left());
+        appendFunctionArg(left(), sb);
```

Comment:
The `instanceof` check must be kept, because `Literal->LeafExpression->Expression` so doesn’t inherit from `NamedExpression` and the `Expressions.name(<literal>)` results to the String `Literal`.

Reply:
Changed the code in `Expressions.name()` to include the case of `Literal` and replaced the `appendFunctionArg()` call with `Expressions.name()`.

Thanks!